lua/rustowl/lsp.lua

```diff
@@ -6,14 +6,96 @@ M.get_rustowl_clients = function(filter)
   filter = vim.tbl_deep_extend('force', filter or {}, {
     name = 'rustowl',
   })
-  return vim.lsp.get_clients(filter)
+  ---@diagnostic disable-next-line: deprecated
+  return type(vim.lsp.get_clients) == 'function' and vim.lsp.get_clients(filter) or vim.lsp.get_active_clients(filter)
+end
+
+---@param client vim.lsp.Client
+---@param root_dir string
+---@return boolean
+local function is_in_workspace(client, root_dir)
+  if not client.workspace_folders then
+    return false
+  end
+  for _, dir in ipairs(client.workspace_folders) do
+    if (root_dir .. '/'):sub(1, #dir.name + 1) == dir.name .. '/' then
+      return true
+    end
+  end
+  return false
+end
+
+---Compatibility for a breaking change in Nvim 0.11
+--- @param client vim.lsp.Client
+--- @param method string LSP method name.
+--- @param params table? LSP request params.
+--- @return boolean status indicating if the notification was successful.
+---                        If it is false, then the client has shutdown.
```

Comment:
```suggestion
--- @return boolean status indicating if the notification was successful. If it is false, then the client has shutdown.
```
lua-language-server doesn't handle multi-line documentation fields, sadly